app/decorators/application_choice_export_decorator.rb

```diff
@@ -1,10 +1,11 @@
 class ApplicationChoiceExportDecorator < SimpleDelegator
   def gcse_qualifications_summary
-    gcses = %i[maths_gcse english_gcse science_gcse]
-    summary_string = gcses
-      .map { |gcse| application_form.send(gcse) }
+    required_subjects = ApplicationQualification::REQUIRED_GCSE_SUBJECTS.reverse
```

Comment:
Why do we need reverse here?

Reply:
The previous code iterated over subjects in the order `maths`, `english`, `science` which is the reverse of `REQUIRED_GCSE_SUBJECTS` but it may be simpler to invert the sort block to
```ruby
      .sort { |a, b| required_subjects.index(b.subject) <=> required_subjects.index(a.subject) }
```